src/App.tsx

```diff
@@ -286,17 +288,19 @@ export const App = ({ shortUrl, user }: Props) => {
       ) : (
         <ul className={commentContainerStyles}>
           {comments.map(comment => (
-            <CommentContainer
-              key={comment.id}
-              comment={comment}
-              pillar="news"
-              shortUrl={shortUrl}
-              onAddComment={onAddComment}
-              user={user}
-              threads={filters.threads}
-              commentBeingRepliedTo={commentBeingRepliedTo}
-              setCommentBeingRepliedTo={setCommentBeingRepliedTo}
-            />
+            <li>
+              <CommentContainer
+                key={comment.id}
```

Comment:
Another reason to not have the wrapper is this `key` prop won't need to be lifted

Reply:
good catch :)